rendering_styles/LightRS.render.xml

```diff
@@ -1006,15 +1007,46 @@
 			</case>
 			
 			<switch textColor="#654321" textWrapWidth="25" textHaloRadius="$textHaloRadiusDay">
-				<switch minzoom="12" tag="natural" value="peak" textItalic="true" textOrder="80">
-					<case maxzoom="13" nameTag="ele" textSize="11"/>
-					<case maxzoom="14" nameTag="" nameTag2="ele" textSize="12"/>
-					<case maxzoom="14" nameTag="ele" textSize="12"/>
-					<case maxzoom="16" nameTag="" nameTag2="ele" textSize="13"/>
-					<case maxzoom="16" nameTag="ele" textSize="13"/>
-					<case minzoom="17" nameTag="" nameTag2="ele" textSize="16"/>
-					<case minzoom="17" nameTag="ele" textSize="16"/>
-				</switch>
+				<case minzoom="12" tag="natural" value="peak" textOrder="80">
+					<case maxzoom="12" textSize="11" intersectionMargin="15">
+						<case eleUnits="feet" nameTag="ele_feet"/>
+						<case nameTag="ele"/>
+					</case>
+					<case minzoom="13" maxzoom="13">
+						<case eleUnits="feet">
+							<case nameTag="" nameTag2="ele_feet" textSize="11"/>
+							<case nameTag="ele_feet" textSize="12"/>
+						</case>
+						<case nameTag="" nameTag2="ele" textSize="11"/>
+						<case nameTag="ele" textSize="12"/>
+					</case>
+					<case minzoom="14" maxzoom="14">
```

Comment:
Too much copy-paste 

You can repalce from 					<case minzoom="14" maxzoom="14">
```
<switch  minzoom="14">
               <case eleUnits="feet" nameTag="" nameTag2="ele_feet"/>
               <case eleUnits="feet" nameTag="ele_feet"/>
               <case nameTag="" nameTag2="ele"/>
               <case nameTag="ele" />
                <apply>
                       <case minzoom="14" maxzoom="15" textSize="13"/>
                       <case minzoom="15" maxzoom="16" textSize="13"/>
                        ...
                </apply>
</switch>
```